contracts/Gateway.sol

```diff
@@ -65,12 +67,16 @@ contract Gateway is
         ReceiverWithdraw[] calldata _receivers,
         uint256 _feeAmount
     ) external payable {
+        if (_feeAmount != feeAmount)
```

Comment:
this condition is not good. `_feeAmount` must be >= and `feeAmount` should be renamed to `minFeeAmount`